advanced/Scripts/piholeCheckout.sh

```diff
@@ -54,6 +54,8 @@ fetch_checkout_pull_branch() {
   # Set the reference for the requested branch, fetch, check it put and pull it
   cd "${directory}"
   git remote set-branches origin "${branch}" || return 1
+  git stash --all --quiet &> /dev/null || return 1
+  git clean --force -d || return 1
```

Comment:
I think we should change this into
```
git stash --all --quiet &> /dev/null || true
git clean --force -d || true
```
since I think `stash --all` and `clean --force` might fail for already clean directories.